src/data/polynomial/algebra_map.lean

```diff
@@ -111,10 +111,37 @@ variables {R A}
 
 theorem aeval_def (p : polynomial R) : aeval x p = eval₂ (algebra_map R A) x p := rfl
 
+@[simp] lemma aeval_zero : aeval x (0 : polynomial R) = 0 :=
+eval₂_zero _ _
```

Comment:
`simp` is already able to prove this: the term would be `alg_hom.map_zero (aeval x)` (and similarly for `aeval_add`, `aeval_mul`, `aeval_one` and `aeval_nat_cast`). I'm OK with using these as abbreviations, but better as abbreviations for `alg_hom.map_...` than `eval₂_...`.